packages/nvidia_nat_adk/tests/test_adk_tool_wrapper.py

```
@@ -194,6 +202,105 @@ async def test_google_adk_tool_wrapper_nested_function(mock_function_tool):
+    class LegacyField:
+        annotation = int
+        outer_type_ = int
+        required = False
+        default = 7
+        default_factory = None
+
+    class LegacyInput:
+        __fields__ = {
+            "required_value": type("RequiredField", (), {"annotation": str, "outer_type_": str, "required": True})(),
+            "optional_value": LegacyField(),
+        }
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)